src/api/dataExport/ScheduleDataExport.ts

```diff
@@ -0,0 +1,70 @@
+/* Import Dependencies */
+import axios from 'axios';
+import KeycloakService from 'app/Keycloak';
+
+/* Import Exceptions */
+import { PostException } from 'app/Exceptions';
+
+
+/**
+ * Function for posting a scheduling job for a data export based upon the provided key and value
+ * @param dataExportKey The key of the property to execute the data export for
+ * @param dataExportValue The value which should be used together with the key to create the data export
+ * @returns Boolean indicating if the download request was succesfull
+ */
+const ScheduleDataExport = async ({ dataExportKey, dataExportValue }: { dataExportKey?: string, dataExportValue?: string }) => {
+    let success: boolean = false;
+
+    if (dataExportKey && dataExportValue) {
+        const token = KeycloakService.GetToken();
+
+        const exportJob: {
+            data: {
+                type: 'export-job',
+                attributes: {
+                    searchParams: {
+                        inputField: string,
+                        inputValue: string
+                    }[],
+                    targetType: string,
+                    exportType: string
+                }
+            }
+        } = {
+            data: {
+                type: 'export-job',
+                attributes: {
+                    searchParams: [
+                        {
+                            inputField: dataExportKey,
+                            inputValue: dataExportValue
+                        }
+                    ],
+                    targetType: 'https://doi.org/21.T11148/894b1e6cad57e921764e',
```

Comment:
Can you pull this from another variable? I think you should have this somewhere

Also, `exportType` will eventually support a few options, might make more sense to store it as a variable.

Reply:
I added a dropdown for the exportType, but how do you mean pull the targetType from another variable? Its not directly linked to a digital specimen or something so I don't have anything to trace that down

Reply:
We will discuss this in a refinement because the fdo type can in the best case come from the schema